Makefile

```diff
@@ -162,7 +162,13 @@ all-tests: eunit test
 docker:
 	@docker build . -t aeternity/epoch:local
 
-smoke-test: system-test-deps docker
+docker-clean:
+	@docker image prune -f
```

Comment:
You may want to do the following to cleanup any system test containers and network:
```
docker rm -f $(docker ps -a | grep 'aeternity/epoch' | cut -d ' ' -f 1)
docker network prune -f
```